client/.storybook/stories/Table.stories.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import { Box } from 'grommet'
+import { Table } from 'components/Table'
+import samples from '../data/samples.js'
+
+const columns = [
+  { Header: 'Technologies', accessor: 'technologies' },
+  { Header: 'Diagnosis', accessor: 'diagnosis' },
+  { Header: 'Disease Timing', accessor: 'disease_timing' }
+]
+
+export default {
+  title: 'Components/Table',
+  args: { columns, data: samples }
```

Comment:
Rather than repeating the same sample 10x in `samples.js`, couldn't you just do something like:

```suggestion
  args: { columns, data: Array(10).fill(sample) }
```

Although actually it'd be nice to do >10 times to see a second page on the table. I think there might be an off by one error in it somewhere because there's two pages shown for only the ten items and they don't change back and forth:

![pagination](https://user-images.githubusercontent.com/4026833/134238781-5e700fe2-ab22-4baa-ab18-302bae920cf0.png)

Reply:
Im going to just unique the samples data more.. i wanted something that would let me show off the filtering anyway

Reply:
Not an off by one error but I did change the getLastOffset helper I added to more accurately handle that